.mergify.yml

```diff
@@ -343,15 +349,18 @@ pull_request_rules:
               - "status-success=mod-check"
               - "status-success=lint-extras"
               - "#changes-requested-reviews-by=0"
-              - "status-success=ci/centos/k8s-e2e-external-storage/1.25"
               - "status-success=ci/centos/k8s-e2e-external-storage/1.26"
               - "status-success=ci/centos/k8s-e2e-external-storage/1.27"
-              - "status-success=ci/centos/mini-e2e-helm/k8s-1.25"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.28"
+              - "status-success=ci/centos/k8s-e2e-external-storage/1.29"
               - "status-success=ci/centos/mini-e2e-helm/k8s-1.26"
               - "status-success=ci/centos/mini-e2e-helm/k8s-1.27"
-              - "status-success=ci/centos/mini-e2e/k8s-1.25"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.28"
+              - "status-success=ci/centos/mini-e2e-helm/k8s-1.29"
               - "status-success=ci/centos/mini-e2e/k8s-1.26"
               - "status-success=ci/centos/mini-e2e/k8s-1.27"
+              - "status-success=ci/centos/mini-e2e/k8s-1.28"
```

Comment:
https://github.com/ceph/ceph-csi/blob/45178c2a905e184f439fbc427609774d27a2c227/.mergify.yml#L151-L205

is not checking 1.26 for devel branch is that expected?

Reply:
That's how the config is currently, I am not able to find the commit that removed 1.26 from that list.
do you want me to add that in this pr ?

Reply:
will revisit the mergify later, not required for now in that case